### ISSUE TITLE
Changing "Farm Account" heading

### ISSUE THREAD

**Issue by aaronhamilton** [brianlala/AutoSPInstaller#69]
Avoid confusion between Farm Admin and Farm Service accounts by changing the title of "Farm Account" to "Farm Service Account".

It seems obvious to seasoned admins, but for those of us with less experience it can slip by undetected ;-)
I put my installer account in here and voila it became my farm service account! :-o

![2020-10-09_19-58-48](https://user-images.githubusercontent.com/22530324/95644957-83eb5480-0a88-11eb-87fb-80fe9be6b76a.png)

**Comment by brianlala** [comment]
Hey Aaron!

AutoSPInstaller Online actually has its own repo: https://github.com/IvanJosipovic/AutoSPInstallerOnlineGithubPage

However... I don't know that there will be much traction here... the project is essentially in maintenance mode with basically only critical updates being made AFAIK. 

Also IMHO this one is debatable :) the next line even states "Add to local admin during setup" - which would not make sense if we were dealing with the installer account  - it needs to be (manually) granted local admin first, prior to any setup activities.

Cheers
Brian

**Comment by aaronhamilton** [comment]
I agree there were clues, and ultimately it was my own fault for not realizing before I ran the install. I was proposing a minor point of clarity to avoid future errors. I defer to the powers that be :-) 

Thanks
Aa
